.github/workflows/main.yml

```diff
@@ -1,7 +1,7 @@
 name: CI
 
 env:
-  LLVM_VERSION_WINDOWS: 9.0.0
+  GITHUB: true
```

Comment:
maybe just use github's default env var? https://help.github.com/en/actions/configuring-and-managing-workflows/using-environment-variables#default-environment-variables

Reply:
Oh man I was looking for these, couldn't find them. Thanks!